app/controllers/api/v2/site_statuses_controller.rb

```diff
@@ -7,6 +7,10 @@ def update
         site_status = authorize SiteStatus.find(params[:id])
         site_status.update site_status_params
 
+        if site_status.no_vacancies?
```

Comment:
do you need to check if it has changed? otherwise for example if there is a course that already have no vacancies and the `status` or `publish` field get changes this would also send out the notification